tutorials/plot_brainstorm_auditory.py

```diff
@@ -219,10 +217,7 @@
 # The epochs overlapping with annotated bad segments are also rejected by
 # default. To turn off rejection by bad segments (as was done earlier with
 # saccades) you can use keyword ``reject_by_annotation=False``.
-picks = mne.pick_types(raw.info, meg=True, eeg=False, stim=False, eog=True,
-                       exclude='bads')
-
-epochs = mne.Epochs(raw, events, event_id, tmin, tmax, picks=picks,
+epochs = mne.Epochs(raw, events, event_id, tmin, tmax, picks=['meg', 'eog'],
```

Comment:
when do we use `('meg', 'eog')` and when `[..]`?

There's a major difference in one or the other in our case, but I found this guy somewhere up there and `(..)` caught my attention. 

```py
epochs = mne.Epochs(raw, events, event_id, tmin, tmax, picks=('meg', 'eeg'),
```
One, the other or both are fine with me (so you can just ignore this)

Reply:
We never use mutable defaults, to tuple in defaults. In other bits of code, does not make much difference.